deps/src/test/java/tests/unit/com/microsoft/azure/sdk/iot/deps/serializer/MethodParserTest.java

```diff
@@ -579,7 +583,7 @@ private static final TestMethod createMethodRequestWithConnectTimeout(
         return testMethod;
     }
 
-    private static final TestMethod createMethodResponse(Integer status, String payload, boolean wrapPayloadAsString, Object expectedPayload)
```

Comment:
no point in declaring private static methods as final, they cannot be overridden anyway